### PR TITLE
Create troubleshooting section in docs and stub out pages

### PR DESCRIPTION
**Pull request by goosemania**
closes #2853
https://pulp.plan.io/issues/2853

### CODE REVIEW

File: docs/troubleshooting/index.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Troubleshooting
+===============
+
+Common problems and how to identify and deal with them can be found here,
+as well as how to file a bug if your issue was not covered.
+
+.. toctree::
```
Comment (goosemania) [inline docs/troubleshooting/index.rst]: I am not sure whether to show `toctree` on the page or just on the sidebar (aka add `:hidden:` here)

File: docs/troubleshooting/index.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+   common-problems
+   file-bug
+
+In case of any questions feel free to reach Pulp community and developers via mailing list or IRC.
+
+* Usage discussions
+
+  * pulp-list@redhat.com
+  * #pulp channel on a freenode
```
Comment (werwty) [inline docs/troubleshooting/index.rst]: s/a freenode/Freenode/

File: docs/troubleshooting/index.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+* Development discussions
+
+  * pulp-dev@redhat.com
+  * #pulp-dev channel on a freenode
```
Comment (werwty) [inline docs/troubleshooting/index.rst]: Same here: s/a freenode/Freenode/

File: docs/troubleshooting/index.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Common problems and how to identify and deal with them can be found here,
+as well as how to file a bug if your issue was not covered.
+
+.. toctree::
+   :maxdepth: 2
+
+   common-problems
+   file-bug
+
+In case of any questions feel free to reach Pulp community and developers via mailing list or IRC.
```
Comment (werwty) [inline docs/troubleshooting/index.rst]: s/reach/reach out to/

File: docs/troubleshooting/index.rst (hunk 1 above)
Comment (werwty) [inline docs/troubleshooting/index.rst]: I think having a toctree here is fine

**Pull request merged**